metadata/src/test/java/org/apache/kafka/timeline/BaseHashTableTest.java

```diff
@@ -121,4 +121,19 @@ public void testExpansion() {
             assertEquals(Integer.valueOf(i), table.baseRemove(Integer.valueOf(i)));
         }
     }
+
+    @Test
+    public void testExpectedSizeToCapacity() {
```

Comment:
As it was a bug, maybe we should check all available numbers. For example:
```java
        IntStream.range(Integer.MIN_VALUE, Integer.MAX_VALUE).forEach(i -> {
            int result = BaseHashTable.expectedSizeToCapacity(i);
            assertTrue(result >= 2);
            assertTrue(result <= BaseHashTable.MAX_CAPACITY);
            assertEquals(0, result % 2);
        });
```

Reply:
Hmm... I think testing 4 billion values is a bit overkill here :)

I added some more test cases, though... Integer.MIN_VALUE, Integer.MAX_VALUE - 1, etc.